template/_.env.local.ejs

```diff
@@ -0,0 +1,3 @@
+# ENV vars here override .env when running locally
+# DATABASE_URL="postgresql://postgres@localhost:5432/<%= name -%>_dev?schema=public"
```

Comment:
Despite this being commented out, should we not follow the same pattern as **prisma** env `DATABASE_URL`:
`DATABASE_URL="postgresql://<%= db.dev.user %><%= db.dev.password %>@<%= db.dev.host %>:<%= db.dev.port %>/<%= db.dev.name %>?schema=public"`

Or is this confusing since this is only necessary to override the **primsa** env `DATABASE_URL`?

Reply:
yeah, we should. we may not need it at all actually, but for now let's at least have the correct values.

Reply:
```suggestion
# DATABASE_URL="postgresql://<%= db.dev.user %><%= db.dev.password %>@<%= db.dev.host %>:<%= db.dev.port %>/<%= db.dev.name %>?schema=public"
```